x-pack/plugins/cases/public/components/all_cases/table_filters.test.tsx

```diff
@@ -257,4 +309,459 @@ describe('CasesTableFilters ', () => {
       expect(onCreateCasePressed).toHaveBeenCalledWith();
     });
   });
+
+  describe('toggle type custom field filter', () => {
```

Comment:
Most of the tests for this feature are here. This way we test the whole integration with the table